pkg/controller/flink/task_manager_controller_test.go

```diff
@@ -136,7 +136,7 @@ func TestTaskManagerHACreateSuccess(t *testing.T) {
 			"high-availability.cluster-id: app-name-"+hash+"\n"+
 			"taskmanager.host: $HOST_IP\n",
 			common.GetEnvVar(deployment.Spec.Template.Spec.Containers[0].Env,
-				"OPERATOR_FLINK_CONFIG").Value)
+				"FLINK_PROPERTIES").Value)
```

Comment:
Shall we continue to test for `OPERATOR_FLINK_CONFIG` being populated here?

Reply:
Added the check.